openfisca_us/variables/gov/states/il/tax/income/personal_exemptions/il_tax_unit_claimable.py

```diff
@@ -0,0 +1,9 @@
+from openfisca_us.model_api import *
+
+
+class il_tax_unit_claimable(Variable):
+    value_type = bool
+    entity = Person
+    label = "Whether a person can be claimed as a dependent by another filer"
```

Comment:
We don't need this. If it's important to the computations, we can put it here: https://github.com/PolicyEngine/openfisca-us/tree/master/openfisca_us/variables/household/demographic/tax_unit